### PR TITLE
[MIG] [13.0] mgmtsystem_hazard

### PR DESCRIPTION
**Pull request by flachica**
@guadaltech @NachoAlesLopez Please, can you take a look to this?

### CODE REVIEW

File: mgmtsystem_hazard/__manifest__.py [hunk 1]
```diff
@@ -13,7 +13,6 @@
         "security/ir.model.access.csv",
         "security/mgmtsystem_hazard_security.xml",
         "views/mgmtsystem_hazard.xml",
-        "views/mgmtsystem_hazard_hazard.xml",
```
Comment (dreispt) [inline mgmtsystem_hazard/__manifest__.py]: Why is this file being removed?

Comment (flachica) [reply mgmtsystem_hazard/__manifest__.py]: The precomit script warned me that there was duplication of Id's. Then I checked that views/mgmtsystem_hazard_hazard.xml and views/mgmtsystem_hazard_origin.xml were identical files

**Review by dreispt — APPROVED** [review]

**Review by max3903 — APPROVED** [review]

File: mgmtsystem_hazard/models/mgmtsystem_hazard.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MgmtsystemHazard(models.Model):
+    """Hazards of the health and safety management system""" ""
```
Comment (NachoAlesLopez) [inline mgmtsystem_hazard/models/mgmtsystem_hazard.py]:
```suggestion
    """Hazards of the health and safety management system"""
```
Remove this string.

**Review by NachoAlesLopez — COMMENTED** [review]
Functional Review: LGTM

File: mgmtsystem_hazard/models/mgmtsystem_hazard.py (hunk 2 above)
Comment (flachica) [reply mgmtsystem_hazard/models/mgmtsystem_hazard.py]: Resolved

**Comment by flachica** [comment]
@NachoAlesLopez @max3903 Merged with hazard_risk maked but wait. I'll make it multi-company asap

**Review by NachoAlesLopez — APPROVED** [review]
OK. Thank you for your work!

**Comment by flachica** [comment]
@NachoAlesLopez @max3903 I have included the multi company. Is it necessary to warn of this?

**Pull request merged**